### DISCUSSION TITLE
Meilisearch integration

### DISCUSSION THREAD

**Discussion by santoshgistto** [pocketbase/pocketbase#4092] [Q&A]
Is it possible to integrate https://www.meilisearch.com/ ?

Thank you

**Comment by daleoflaherty** [comment]
These external search-indexes-as-APIs are best used just like that, as external APIs. Manage them over http according to the needs of your app and they often want to be exposed directly to the user rather than proxied through the backend. Host it in a container on the same host as your pocketbase server or use their cloud offering.

PB would interact with it just like any other backend would: You could spawn jobs for adding things to the index / performing updates or you could add them directly in the record hooks. Pass the short-lived search api keys to the client as a cookie alongside their auth request or provide a dedicated endpoint. Hit the search index directly from the client. Plug in an algolia style search widget to get started if they don't offer their own and you're unsure of how to go about the autocomplete, they usually try to have compatible apis.

**Reply by santoshgistto** [reply]
Thank you
